device/std.self.json

```diff
@@ -0,0 +1,28 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:azure:common:SelfDescribing;1",
+  "@type": "Interface",
+  "displayName": "IDispatch",
+  "contents": [
+    {
+      "@type": "Command",
+      "name": "GetTargetModel",
```

Comment:
I'm not sure on the term "target" for self-describing device models. Why target? It hopes to live up to it, but might not?

Reply:
it has been discussed by the working group, and these are the param names we agreed on.

Reply:
So we're done taking feedback then? And you can't tell me what "target" is supposed to mean?

Reply:
there are two models in the solution, 1) the SDD interface and 2) the customer interface. 
1) is in the public repo 2) is created by the user and served from the device.

We decided to call 2) targetModel and targetModelId. 

These names have been decided by our LT, but that doesn't mean we are not taking additional feedback.

Reply:
Since when does LT make detailed naming decisions that engineering can't challenge? That is just dumb.

I still don't have an answer to my question about what "target" refers to. It isn't obvious/intuitive to me, so I'm just asking what the thought is behind it. Maybe it'll make sense to me.

Reply:
I'll go talk to Stefan.